pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/ConfigUtil.java

```diff
@@ -53,6 +54,18 @@ public static String getStringOrElse(Config cfg, String key, String defaultValue
         return cfg.hasPath(key) ? cfg.getString(key) : defaultValue;
     }
 
+    public static Optional<String> getString(Config cfg, String key) {
+        if (cfg.hasPathOrNull(key)) {
+            if (cfg.getIsNull(key)) {
+                return Optional.empty();
+            } else {
+                return Optional.of(cfg.getString(key));
+            }
+        } else {
+            return Optional.empty();
+        }
+    }
+
```

Comment:
Added as the existing `getStringOrElse` and `getStrIfPresent` methods will throw an error if the specified key is explicitly set to `null`.

Reply:
Isn't it overcomplicated? Config also has the method `hasPath` that covers the external switch statement.

I think this will do the same:
```
public static Optional<String> getString(Config cfg, String key) {
    if (!cfg.hasPath(key)) {
        return Optional.empty();
    }

    return Optional.of(cfg.getString(key));
}
```

Reply:
No, it does not- In that case, `cfg.getString()` will throw an exception in the following case:
```
{
  myKey = null
}
```

In this case, `hasPath()` returns `true`, but `getString()` will throw due to the null value.